addons/website_blog/controllers/main.py

```diff
@@ -164,7 +169,101 @@ def tags_list(tag_ids, current_tag):
             'blog_url': blog_url,
             'date': date_begin,
         }
-        response = request.website.render("website_blog.blog_post_short", values)
+
+        if feed_type in ['atom','rss']:
+            response = self._blog_feed(blog, domain, feed_type)
+        else:
+            response = request.website.render("website_blog.blog_post_short", values)
+        return response
+
+    def _blog_feed(self, blog, domain, feed_type):
+        """ RSS/ATOM
+        
+        Blog Detail : fg = FeedGenerator()
+
+        - Id : Blog id (URL)
+        - Title : Blog name
+        - Link : Blog LINK (URL)
+        - Description : Blog Description
+        - Lastbuilddate : Update date (last modification date in the blog) "This will set both ATOM:updated and RSS:lastBuildDate."
+        - Language : blog language
+            ------
+            Blog Post Detail : fe = fg.add_entry()
+            ------
+            - Id : Blog Post id (URL)
+            - Title : Blog Post Name
+            - Link : Blog Post Link (URL) OR Rss Link OR Atom Link
+            - Author : Blog Post Name(Name, Email)
+            - Description : Blog Post Subtitle
+            - Update : last modification date in the Blog Post for use ATOM
+            - Published : Create Date of Blog Post for use ATOM
+            - Pubdate : Create Date of Blog Post for use RSS
+            - Content : Blog Post Content (Diasplay Content In ATOM) "The content MUST be suitable for presentation as HTML"
+        """
+        limit = 15
+        cr, uid, context = request.cr, request.uid, request.context
+        user = request.registry['res.users'].browse(cr, uid, uid, context=context)
+        base_url = request.registry['ir.config_parameter'].get_param(cr, uid, 'web.base.url')
+        blog_post_obj = request.registry['blog.post']
+
+        timezone_format = '%a, %d %b %Y %H:%M:%S %z'
```

Comment:
@tde-banana-odoo timezone formate required into python-feedgen library then i added,
but not now necessary then i removed.
because problem fixed into python-feedgen.